eo-runtime/src/main/java/org/eolang/SafeFunc.java

```diff
@@ -47,18 +47,17 @@ class SafeFunc<T> implements Supplier<T> {
     }
 
     @Override
+    @SuppressWarnings({"PMD.AvoidCatchingGenericException", "PMD.AvoidRethrowingException"})
     public T get() {
         try {
             return this.origin.call();
         } catch (final InterruptedException ex) {
             Thread.currentThread().interrupt();
-            throw new ExInterrupted();
-        } catch (final ExAbstract ex) {
-            throw ex;
+            throw new ExInterrupted(ex);
             // @checkstyle IllegalCatchCheck (3 line)
         } catch (final RuntimeException ex) {
             throw ex;
-        } catch (final Throwable ex) {
+        } catch (final Exception ex) {
```

Comment:
@c71n93 this seems to be a pretty dangerous change. `Throwable` is of a higher level than `Exception` - we must catch it

Reply:
@yegor256 I made it because of `PMD.AvoidCatchingThrowable` violation. It seemed reasonable to me. As far as I understand, it is pointless to catch `Error`, since it indicates irrecoverable situation. According to https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/lang/Error.html "An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch."

Reply:
@yegor256 we catch `RuntimeException` two lines above, so here we can catch either `Exception` either `Throwable` - it would be the same.